app/src/main/java/com/capstone/event_finder/adapters/PhotoAdapter.java

```diff
@@ -59,5 +66,14 @@ public void bind(Photo post) {
                 Glide.with(context).load(image.getUrl()).into(ivAlbumPhotoItem);
             }
         }
+
+        private void goToPhotoDetailsActivity(View v) {
```

Comment:
✅ Great job continuing to follow up with breaking code into methods!

Reply:
> Looks good, for these animation related PRs it would be great to see a video of what the transitions look like!

Yes! Planning to add a demo to my update later today!